src/main/webapp/WEB-INF/views/UserManagement.jsp

```diff
@@ -0,0 +1,142 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" %>
+    <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+        <html>
+
+        <head>
+            <title>User Management</title>
+            <style>
+                .username.ng-valid {
```

Comment:
see similar comments on header consistency.  Consider using a .tag file to create a consistent header.